tools/devops/automation/scripts/TestResults.Tests.ps1

```diff
@@ -140,12 +140,12 @@ Describe "TestResults tests" {
             New-Item -Path "$testDirectory/TestSummary-prefixintrospection-2" -Name "TestSummary.md" -Value "SummaryE" -Force
             New-Item -Path "$testDirectory/TestSummary-prefixmtouch-3" -Name "TestSummary.md" -Value "SummaryF" -Force
 
-            $labels = "linker;introspection".Split(";")
+            $labels = "linker;introspection;monotouch-test".Split(";")
```

Comment:
why are you doing a `Split` when you could do:
```
$labels = @("linker", "introspection", "monotouch-test")
```

Reply:
Because by the time I'm able to make my powershell changes work, I don't want to touch them again...